examples/placegoose/wrangler.toml

```
@@ -0,0 +1,50 @@
+name = "placegoose"
+compatibility_date = "2024-11-15"
+
+compatibility_flags = [ "nodejs_compat" ]
+
+assets = { directory = "public" }
+
+# Includes .md files in bundle: 
+# https://developers.cloudflare.com/workers/wrangler/configuration/#bundling
+rules = [
+  { type = "Text", globs = ["**/*.md"], fallthrough = true }
+]
+
+# [vars]
+# MY_VAR = "my-variable"
+
+# [[kv_namespaces]]
+# binding = "CACHE"
+# id = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+# [[r2_buckets]]
+# binding = "MY_BUCKET"
+# bucket_name = "my-bucket"
+
+[[d1_databases]]
+binding = "DB"
+database_name = "placegoose-d1"
+database_id = "local-placegoose-d1"
+migrations_dir = "drizzle/migrations"
+
+# [ai]
+# binding = "AI"
+
+[observability]
+enabled = true
+# head_sampling_rate = 1
+
+# The rate limiting API is in open beta.
+# https://www.npmjs.com/package/@hono-rate-limiter/cloudflare
+[[unsafe.bindings]]
+name = "RATE_LIMITER"
+type = "ratelimit"
+# An identifier you define, that is unique to your Cloudflare account.
+# Must be an integer.
+namespace_id = "1001"
```

just realized: it could be good to use a namespace id that's different than the readme's default, just to be safe. but i haven't looked up what the rules are for namespace ides

```suggestion
namespace_id = "6666"
```